packages/css-library/src/stylesheets/base/_va.scss

```diff
@@ -351,4 +351,505 @@ article > h1 {
     font-weight: $font-weight-normal;
     line-height: $lead-line-height;
   }
+}
+
+// TODO: Integrate USWDS and elements/_typography.scss
+// and eliminate this declaration
+.usa-content {
+  max-width: $text-max-width;
+}
+
+//====================
+// Content callout boxes
+// ===================
+
+.feature {
```

Comment:
I don't think we included any of these styles in `_va.scss` because at the time the assumption was that they would be replaced with web components or removed completely.

Because our timeline is condensed, I'm just adding it all here for now so that we can make adjustments to them if needed.